yaptide/celery/tasks.py

```diff
@@ -32,7 +28,8 @@ def run_shieldhit(self, param_dict: dict, raw_input_dict: dict):
         # digest dictionary with project data (extracted from JSON file)
         # and generate SHIELD-HIT12A input files
         conv_parser = get_parser_from_str("shieldhit")
-        run_parser(parser=conv_parser, input_data=raw_input_dict, output_dir=tmp_output_path)
+        run_parser(parser=conv_parser, input_data=raw_input_dict,
```

Comment:
why this change ? we should support lines of 120 characters

Reply:
It was autocorrect by IDE, rechanged to original